loom/common/xml/tinyxml2.h

```diff
@@ -37,6 +39,8 @@ distribution.
 	gcc: g++ -Wall tinyxml2.cpp xmltest.cpp -o gccxmltest.exe
 */
 
+#define TIXML2_MEM_ALIGN_ENABLED 0

```

Comment:
I thought it was necessary, but it didn't turn out to be, though I kept it in as an option. Should we remove it as additional cruft?